client/src/App.js

```diff
@@ -1,52 +1,98 @@
-import React from "react";
+import React, {Component} from "react";
 import "./App.css";
 import Search from "./components/Search";
+import Timetable from "./components/TimeTable";
+
+
+class App extends Component {
+  constructor(props)
+  {
+    super(props);
+    this.state = {
+      isRoot: 1,
+      isPreview: 0
```

Comment:
there is a boolean type in JavaScript. Unless these were meant to be integers, consider changing them to bool

Reply:
Okay